hoomd/md/PotentialExternalGPUKernel.cu.inc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2009-2022 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#include "hoomd/md/PotentialExternalGPU.cuh"
+#include "hoomd/md/EvaluatorExternal@_evaluator@.h"
+
+namespace hoomd { namespace md { namespace kernel {
+template
+__attribute__((visibility("default")))
+hipError_t
+gpu_cpef<EvaluatorExternal@_evaluator@>(const external_potential_args_t& external_potential_args,
```

Comment:
In other places in this PR, we have changed the abbreviated names to more verbose versions. In this case, the changed name would be `gpu_compute_potential_external_forces`.

Reply:
In other places, I didn't change any names - I removed an abbreviated template name and started using the full name. The original code base used `gpu_cpef` as the original full name for this potential for unknown reasons.